frontend/packages/dev-console/src/utils/__tests__/pipeline-test-data.ts

```diff
@@ -244,7 +244,9 @@ export const constructPipelineData = {
   pipelineRuns: [
     specificPipelineData.pipelineRuns[DataState.SUCCESS],
     specificPipelineData.pipelineRuns[DataState.IN_PROGRESS],
-    specificPipelineData.pipelineRuns[DataState.CANCELLED],
+    specificPipelineData.pipelineRuns[DataState.CANCELLED1],
+    specificPipelineData.pipelineRuns[DataState.CANCELLED2],
+    specificPipelineData.pipelineRuns[DataState.CANCELLED3],
```

Comment:
Don't think we needed to add `CANCELLED2` and `CANCELLED3` to this test data, but it appears to have not adjusted the intent of the test (which is get the latest run). I think it's okay to leave this change.